opendevin/controller/agent_controller.py

```diff
@@ -76,6 +76,7 @@ def __init__(
         """
         self._step_lock = asyncio.Lock()
         self.id = sid
+        print('\n\n\n\\ agent =', agent)
```

Comment:
Is this intentional? Maybe it should be a logging command or removed?

Reply:
This is my bad habit of using printing as debugging statements :) Will remove